app/src/androidTest/java/org/mozilla/fenix/ui/DownloadTest.kt

```diff
@@ -50,49 +61,288 @@ class DownloadTest {
     fun testDownloadPrompt() {
-        val defaultWebPage = TestAssetHelper.getDownloadAsset(mockWebServer)
+        fileName = "web_icon.png"
 
         navigationToolbar {
-        }.enterURLAndEnterToBrowser(defaultWebPage.url) {
-            mDevice.waitForIdle()
-        }
-
+        }.enterURLAndEnterToBrowser(testPage.toUri()) {
+        }.clickDownloadLink(fileName) {
+            verifyDownloadPrompt(fileName)
+        }.clickDownload {
+            verifyDownloadNotificationPopup()
+        }.clickOpen("image/png") {}
         downloadRobot {
-            verifyDownloadPrompt()
-        }.closePrompt {}
+            verifyPhotosAppOpens()
+        }
     }
 
     @Test
-    fun testDownloadNotification() {
-        val defaultWebPage = TestAssetHelper.getDownloadAsset(mockWebServer)
+    fun testCloseDownloadPrompt() {
+        fileName = "smallZip.zip"
 
         navigationToolbar {
-        }.enterURLAndEnterToBrowser(defaultWebPage.url) {
-            mDevice.waitForIdle()
+        }.enterURLAndEnterToBrowser(testPage.toUri()) {
+        }.clickDownloadLink(fileName) {
+            verifyDownloadPrompt(fileName)
+        }.closePrompt {
+        }.openThreeDotMenu {
+        }.openDownloadsManager {
+            verifyEmptyDownloadsList()
         }
+    }
 
-        downloadRobot {
-            verifyDownloadPrompt()
+    @Test
+    fun testDownloadCompleteNotification() {
+        fileName = "smallZip.zip"
+
+        navigationToolbar {
+        }.enterURLAndEnterToBrowser(testPage.toUri()) {
+        }.clickDownloadLink(fileName) {
+            verifyDownloadPrompt(fileName)
         }.clickDownload {
             verifyDownloadNotificationPopup()
         }
-
         mDevice.openNotification()
         notificationShade {
             verifySystemNotificationExists("Download completed")
         }
-        // close notification shade before the next test
-        mDevice.pressBack()
+    }
+
+    @SmokeTest
+    @Test
+    fun pauseResumeCancelDownloadTest() {
+        fileName = "1GB.zip"
+
+        navigationToolbar {
+        }.enterURLAndEnterToBrowser(testPage.toUri()) {
+        }.clickDownloadLink(fileName) {
+            verifyDownloadPrompt(fileName)
+        }.clickDownload {}
+        mDevice.openNotification()
+        notificationShade {
+            expandNotificationMessage()
+            clickSystemNotificationControlButton("Pause")
+            clickSystemNotificationControlButton("Resume")
+            clickSystemNotificationControlButton("Cancel")
+            mDevice.pressBack()
+        }
+        browserScreen {
+        }.openThreeDotMenu {
+        }.openDownloadsManager {
+            verifyEmptyDownloadsList()
+        }
+    }
+
+    @SmokeTest
+    @Test
+        /* Verifies downloads in the Downloads Menu:
+          - downloads appear in the list
+          - deleting a download from device storage, removes it from the Downloads Menu too
+        */
+    fun manageDownloadsInDownloadsMenuTest() {
```

Comment:
It seems weird that we're using a local asset here now while the other tests are using remote assets. Would it make sense for this test to just use the Cloud Storage?

Reply:
done

Reply:
Do you need me to upload a file?

Reply:
No, thanks. I asked Isabel to do it on Friday.